README.md

```diff
@@ -185,13 +212,22 @@ Try install [Microsoft Visual C++ Redistributable](https://aka.ms/vs/16/release/
 
 ## Build ZENO
 
-- Linux
+- Linux(Previous)
 
 ```bash
 cmake -B build
 make -C build -j8
 ```
 
+- Ubuntu 20.04
+
+```bash
+# clone ZENO
+git clone git@github.com:zenustech/zeno.git
```

Comment:
let's always use https here, many people (including zhxx) don't have ssh key for git.

Reply:
Got it! I will change it to https.

Reply:
I have ssh key, just not using it!